ibm/service/kubernetes/resource_ibm_container_vpc_worker_pool.go

```diff
@@ -232,6 +239,33 @@ func ResourceIBMContainerVPCWorkerPoolValidator() *validate.ResourceValidator {
 
 func resourceIBMContainerVpcWorkerPoolCreate(d *schema.ResourceData, meta interface{}) error {
 
+	if ioc, ok := d.GetOk("import_on_create"); ok && ioc.(bool) {
```

Comment:
How safe is `ioc.(bool)` ? Will it panic if ioc is not a bool?

Reply:
it would panic!

but that would mean that 			
```
"import_on_create": {
				Type:             schema.TypeBool,
```

doesnt really have meaning :)

this supposed to ensure this. I will dig into this further.

Reply:
oh and answer to question , ioc in its form supposed to be an interface.  .(bool) converts it to bool. and than it has value based on the content.